### PR TITLE
Pipeline now correctly processes donations data

### PR DESCRIPTION
**Pull request by dcstlouis**
There was an issue with the pipeline process and it wasn't importing donations data. I updated the CSV_HEADERS so they match the headers in the input file.

### CODE REVIEW

**Review by c-simpson — CHANGES_REQUESTED** [review]
What was the file causing problems (an old one, possibly?)  We had the import working pretty well with current files.

File: src/server/api/file_uploader.py [hunk 1]
```diff
@@ -34,12 +34,11 @@ def determine_upload_type(file, file_extension):
         dfs = [pd.read_csv(io.BytesIO(file.stream.read()), encoding='iso-8859-1')]
         file.close()
     else:
-        
         match = re.search('donat', file.filename, re.I)
 
         if match:   # It's a SalesForce Donations file
             validate_import_sfd(file)
-            return
+            dfs = excel_to_dataframes(file)
```
[the remark this answers is not in the document]
Comment (dcstlouis) [reply src/server/api/file_uploader.py:41]: I was having trouble with the donations file; it wasn't importing all the rows. I saw "Row 1000" in the logs and then the stats.

File: src/server/datasource_manager.py [hunk 2]
```diff
@@ -16,11 +16,10 @@ def __clean_csv_headers(header):
...
     'volgisticsshifts': ['Number', 'Place', 'Assignment', 'From date', 'To date', 'Hours'],
-    'salesforcedonations': ['Recurring donor', 'Opportunity Owner', 'Account Name', 'Opportunity ID (18 Digit)',
-                            'Account ID (18 digit)',
-                            'Opportunity Name', 'Stage', 'Fiscal Period', 'Amount', 'Probability (%)', 'Age',
-                            'Close Date', 'Created Date', 'Type', 'Primary Campaign Source',
-                            'Source', 'Contact ID (18 Digit)', 'Primary Contact'],
+    'salesforcedonations': ['Recurring donor - A', 'Opportunity Owner', 'Account Name', 'Opportunity ID 18', 
+                            'Account ID 18', 'Opportunity Name', 'Stage', 'Fiscal Period', 'Amount', 
+                            'Probability (%)', 'Age', 'Close Date', 'Created Date', 'Type', 'Primary Campaign Source',
+                            'Source', 'Contact ID 18', 'Primary Contact'],
```
[the remark this answers is not in the document]
Comment (dcstlouis) [reply src/server/datasource_manager.py:22]: Can we get rid of it?

File: src/server/api/file_uploader.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (c-simpson) [reply src/server/api/file_uploader.py:41]: Here's what I see loading the big file (Salesforce_PAWS Donation (all Time) 2021-06-15.xlsx)


```
127.0.0.1 - - [07/Sep/2021 10:00:19] "GET /api/statistics HTTP/1.1" 200 -
[2021-09-07 10:00:37,646] INFO in file_uploader: Start uploading file: Salesforce_PAWS Donation (all Time) 2021-06-15.xlsx
[2021-09-07 10:00:37,647] INFO in donations_importer: ---------------------- Loading Salesforce_PAWS Donation (all Time) 2021-06-15.xlsx------------------------
C:\Python38\lib\site-packages\openpyxl\styles\stylesheet.py:214: UserWarning: Workbook contains no default style, apply openpyxl's default
  warn("Workbook contains no default style, apply openpyxl's default")
Minimum similarity: 0.93
On expected/got:  Recurring donor / Recurring donor - A
[2021-09-07 10:04:23,660] INFO in donations_importer: Row: 1000
[2021-09-07 10:04:27,212] INFO in donations_importer: Row: 2000
[2021-09-07 10:04:30,935] INFO in donations_importer: Row: 3000
[2021-09-07 10:04:34,781] INFO in donations_importer: Row: 4000
[2021-09-07 10:04:39,240] INFO in donations_importer: Row: 5000
[2021-09-07 10:04:42,465] INFO in donations_importer: Row: 6000
[2021-09-07 10:04:45,498] INFO in donations_importer: Row: 7000
```

and so on ...

Here's what I see when loading the update file  (Salesforce_PAWS Donation_6_14_2021_TO_7_27_2021.xlsx)
Could that have been what you were loading?

```
[2021-09-07 10:10:48,268] INFO in file_uploader: Start uploading file: Salesforce_PAWS Donation_6_14_2021_TO_7_27_2021.xlsx
[2021-09-07 10:10:48,268] INFO in donations_importer: ---------------------- Loading Salesforce_PAWS Donation_6_14_2021_TO_7_27_2021.xlsx------------------------
Minimum similarity: 0.93
On expected/got:  Recurring donor / Recurring donor - A
[2021-09-07 10:10:53,407] INFO in donations_importer: Row: 1000
[2021-09-07 10:10:53,710] INFO in donations_importer: ---------------------------------   Stats  -------------------------------------------------
[2021-09-07 10:10:53,711] INFO in donations_importer: Total rows: 1107 Dupes: 0 Missing contact_id: 156
[2021-09-07 10:10:53,712] INFO in donations_importer: Other integrity exceptions: 0 Other exceptions: 0
127.0.0.1 - - [07/Sep/2021 10:10:53] "POST /api/file HTTP/1.1" 302 -
```




This was on   master 43d02aafd5be04ba47f6ce29e0ab99f81cf2120e

**Comment by dcstlouis** [comment]
Closing

**Pull request closed without merging**